sdk/compiler/daml-extension/src/extension.ts

```diff
@@ -221,7 +221,7 @@ async function startLanguageServers(context: ExtensionContext) {
     (await fileExists(rootPath + "/.envrc")) &&
     vscode.extensions.getExtension("mkhl.direnv") == null;
 
-  if (envrcExistsWithoutExt && gradleSupport) {
+  if (envrcExistsWithoutExt) {
```

Comment:
People use `.envrc` without gradle, this message should not be restricted to that